src/stories/Foundation/Tooltip/Tooltip.stories.tsx

```diff
@@ -1,21 +1,40 @@
+import { useState } from 'react';
 import styled from 'styled-components';
 import { Tooltip } from '.';
 import { Button } from '../../Forms/Button';
 
 export default { component: Tooltip };
 
 const Wrapper = styled.div`
-  margin-top: var(--spacing-32);
+  display: flex;
+  margin-top: var(--spacing-48);
+  justify-content: center;
 `;
 
-const BasicTemplate = () => {
+const Template = ({ alignX, alignY }) => {
+  const [veryCount, setVeryCount] = useState(1);
+
+  const getTooltipText = () => {
+    const veryStr = Array(veryCount).fill('very').join(' ');
+    return `A ${veryStr} cool tooltip! ✨`;
+  };
+
+  const addVery = () => {
```

Comment:
love how you name this function add so it is very easy to see that you add one! 💡